packages/filesystem/src/node/file-change-collection.ts

```diff
@@ -22,7 +22,7 @@ import { FileChange, FileChangeType } from '../common/filesystem-watcher-protoco
  * Changes are normalized according following rules:
  * - ADDED + ADDED => ADDED
  * - ADDED + UPDATED => ADDED
- * - ADDED + DELETED => NONE
+ * - ADDED + DELETED => [ADDED, DELETED]
```

Comment:
It could happen before as well, no? client has to handle such cases in any way, noone guarantees that a file won't be gone by the moment then you try to read it after receiving ADDED event.

Reply:
AFAIC before you wouldn't see any event in that case, right? that's what NONE means here.
so, if there is any code that start reading content on `ADDED`, it will need to handle errors properly in the case the resource is missing. but, just wanted to consider this. I think it would have been buggy anyway to not consider errors with resources anyway.

Reply:
i see, i don't know how to invalidate monaco models without this change

Tests create and delete file very fast, so Monaco never got notified that file was deleted before. Not sure whether it can happen in practice that someone programmatically create and delete files very fast, but then it will lead to same issues.